scripts/ff_ci_pr_build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
Hi @jakirkham, Sure. I think the docs can live in the actual script, though maybe this one should contain a comment that links back to that explictly, ie "For more information...". Also please correct me if I am wrong, but the original script still does not contain any documentation.